ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftSDO",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadC2",
```

Comment:
Do we need that as it was actually promoted?

Reply:
Nope, mentioned that already to James. Will remove for now, although there is still confusion around what the permissions should be. Will leave that confusion with James.